services/scan/src/store.ts

```diff
@@ -404,90 +516,142 @@ export class Store {
     );
   }
 
+  /**
+   * Returns the id of the an unfinished batch if there is one
+   */
+  getOngoingBatchId(): Optional<string> {
+    const ongoingBatchRow = this.client.one(
+      'select id from batches where ended_at is null'
+    ) as { id: string } | undefined;
+
+    return ongoingBatchRow?.id;
+  }
+
   /**
    * Records that batches have been backed up.
    */
-  setScannerAsBackedUp(): void {
-    this.setBackupTimestamp(BackupKey.Scanner);
+  setScannerBackedUp(backedUp = true): void {
+    if (!this.hasElection()) {
+      throw new Error('Unconfigured scanner cannot be backed up.');
+    }
+
+    if (backedUp) {
+      this.client.run(
+        'update election set scanner_backed_up_at = current_timestamp'
+      );
+    } else {
+      this.client.run('update election set scanner_backed_up_at = null');
+    }
   }
 
   /**
    * Records that CVRs have been backed up.
    */
-  setCvrsAsBackedUp(): void {
-    this.setBackupTimestamp(BackupKey.Cvrs);
+  setCvrsBackedUp(backedUp = true): void {
+    if (!this.hasElection()) {
+      throw new Error('Unconfigured scanner cannot export CVRs.');
+    }
+
+    if (backedUp) {
+      this.client.run(
+        'update election set cvrs_backed_up_at = current_timestamp'
+      );
+    } else {
+      this.client.run('update election set cvrs_backed_up_at = null');
+    }
   }
 
   /**
-   * Sets the timestamp of a backup
+   * Gets the timestamp for the last scanner backup
    */
-  private setBackupTimestamp(key: BackupKey): void {
-    this.client.run(
-      'insert or replace into backups (key, value) values (?, current_timestamp)',
-      key
-    );
+  getScannerBackupTimestamp(): Iso8601Timestamp | undefined {
+    const row = this.client.one(
+      'select scanner_backed_up_at as scannerBackedUpAt from election'
+    ) as { scannerBackedUpAt: Iso8601Timestamp } | undefined;
+    return row?.scannerBackedUpAt;
   }
 
   /**
-   * Gets a config value by key.
+   * Gets the timestamp for the last cvr export
    */
-  private getBackupTimestamp(key: BackupKey): Iso8601Timestamp | undefined {
-    const row = this.client.one<[Iso8601Timestamp]>(
-      'select value from backups where key = ?',
-      key
-    ) as { value: Iso8601Timestamp } | undefined;
-    return row?.value;
+  getCvrsBackupTimestamp(): Iso8601Timestamp | undefined {
+    const row = this.client.one(
+      'select cvrs_backed_up_at as cvrsBackedUpAt from election'
+    ) as { cvrsBackedUpAt: Iso8601Timestamp } | undefined;
+    return row?.cvrsBackedUpAt;
+  }
+
+  getBallotsCounted(): number {
+    const r = this.client.one(`
```

Comment:
Use `row` like we do elsewhere.
```suggestion
    const row = this.client.one(`
```